vignettes/pkgdown.Rmd

```diff
@@ -178,4 +180,6 @@ Once your finalized site is built and published on the web, you should publicize
 
 2.  Your repository description on GitHub.
 
+    (`usethis::use_pkgdown_github_pages()` does this for you.)
```

Comment:
If we're going to mention it in the bullet point above, we should mention it here. It's true.